src/handlers/dns/DnsStreamHandler.cpp

```diff
@@ -76,6 +76,9 @@ void DnsStreamHandler::start()
             throw ConfigException("DnsStreamHandler: only_rcode filter contained an invalid/unsupported rcode");
         }
     }
+    if (config_exists("only_dns_sec_response")) {
```

Comment:
needs to be "only_dnssec_response" with no _ between dns and sec

Reply:
done